libs/ledger-live-common/src/families/elrond/helpers/areEarnRewardsEnabled.ts

```diff
@@ -1,12 +1,7 @@
 import { BigNumber } from "bignumber.js";
-import { denominate } from "./denominate";
 import { Account } from "@ledgerhq/types-live";
 
-export const areEarnRewardsEnabled = (account: Account): boolean => {
-  return BigNumber(
-    denominate({
-      input: String(account.spendableBalance),
-      showLastNonZeroDecimal: true,
-    }),
-  ).gte(1);
-};
+import { MIN_DELEGATION_AMOUNT } from "../constants";
+
+export const areEarnRewardsEnabled = (account: Account) =>
+  BigNumber(account.spendableBalance).isGreaterThanOrEqualTo(MIN_DELEGATION_AMOUNT);
```

Comment:
- `account.spendableBalance` is already a BigNumber, you don't need to wrap it
- (correct syntax is `new BigNumber(...)` instead of `BigNumber(...)`)
- This helper should be used in the UI instead of duplicating the logic there
- Is the issue only on LLD, no need for a fix on LLM?